mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1216,6 +1216,7 @@ def LLVM_GlobalOp : LLVM_Op<"mlir.global",
       "StringRef":$name, "Attribute":$value,
       CArg<"uint64_t", "0">:$alignment,
       CArg<"unsigned", "0">:$addrSpace,
+      CArg<"std::optional<StringRef>", "std::nullopt">:$section,
```

Comment:
I guess LLVM dialect fixes need to go through main llvm-project repo...

Reply:
Ok. I will prepare the PR to LLVM

Reply:
looks like we can do the same without any changes to ```mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td```